wsd/DocumentBroker.cpp

```diff
@@ -3405,8 +3404,7 @@ void DocumentBroker::sendRequestedTiles(const std::shared_ptr<ClientSession>& se
                     tilesNeedsRendering.push_back(tile);
                     _debugRenderedTileCount++;
                 }
-                if (tileCache().subscribeToTileRendering(tile, session, now))
-                    beingRendered++;
+                tileCache().subscribeToTileRendering(tile, session, now);
```

Comment:
presumably the return value from subscribeToTileRendering can be dropped again after that, as no more users of it

Reply:
Dropping the beingRendered would have, presumably, also gotten me out of the hole I found myself in during ed8f95c1bc5ac9a2e5c7dd2fb17d7a94f9d8b2b5 trying to get the empty-update tests in place.

I wonder should we continue to "subscribeToTileRendering" + "sendTileCombine" in handleTileCombinedRequest before calling "sendRequestedTiles" which also does those things? Is that a duplication of effort?